apps/search/collections/global_search_results.coffee

```diff
@@ -0,0 +1,22 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+sd = require('sharify').data
+qs = require 'querystring'
+SearchResult = require '../../../models/search_result.coffee'
+
+module.exports = class GlobalSearchResults extends Backbone.Collection
+  model: SearchResult
+
+  url: "#{sd.API_URL}/api/v1/match"
+
+  parse:  (response) ->
+    _.reject response, (item) ->
+      # HACK filter out sensitive results (at the artist's request)
+      JSON.stringify(item).match(/kippenberger|zoe.*leonard|pat.*lipsky/i)
```

Comment:
Thanks for the info 👍 What @joeyAghion is getting at is basically why I wanted to know this.